python/lsst/ts/IntegrationTests/configs/love_stress_test_configs.py

```diff
@@ -0,0 +1,62 @@
+# This file is part of ts_IntegrationTests.
+#
+# Developed for the Rubin Observatory Telescope and Site System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import yaml
+
+from .config_registry import registry
+
+
+# Add the LOVE Stress Test script
+# configurations to the registry.
+
+registry["love_stress"] = yaml.safe_dump(
+    {
+        "location": "love.tu.lsst.org",
```

Comment:
Can you confirm this with Sebastian as this URL is the k8s instance. The bare metal one is `love1.tu.lsst.org`.

Reply:
It seems that either value will work.  I'm going to leave it as is and we'll see how things go when we test it out.